app/retail/views.py

```diff
@@ -200,6 +200,127 @@ def index(request):
     return TemplateResponse(request, 'home/index.html', context)
 
 
+@staff_member_required
+def pricing(request):
+
+    plans= [
```

Comment:
Nice configurability! Should we move this into an import to keep this file a little leaner? Also I'm wondering if we should keep this loaded in memory instead of loading it up for each request

Reply:
@danlipert I do agree with maybe importing. We can actually make a constants.py file and would preferably need cleaning a lot of views. But I didn't get what you mean by keeping this loaded in memory.

Reply:
A single constants file ? where would this reside ? 
I'd be happy to throw that in now or a follow up PR i have coming in a few days

Reply:
I think a follow up PR makes sense while cleaning every such occurence over different views.

Reply:
@SaptakS @thelostone-mc I think a constants file is a good idea in general, but what I meant by my comment was a bit different: Specifically I'm talking about taking the final `context` variable and moving it outside of the `def pricing(request):` handler definition.

I like to declare psuedo "constants" in all capital letters below the imports. Since this one is many lines of code, it makes sense to import it and keep this file tidy. By doing this, the handler gets a bit speedier in that it doesn't have to re-create the `context` for every request, it already has it available in memory to read and return. We get the same functionality but better performance at the cost of a bit of memory usage.